From c565d1458d641a73953a9c322a92aa2b7dc88c96 Mon Sep 17 00:00:00 2001
From: OS2 <mustafaaslan35@gmail.com>
Date: Sun, 18 Oct 2026 13:31:53 +0300
Subject: [PATCH] =?UTF-8?q?debian:=200.5.1+bacak1=20=E2=80=94=20linux-head?=
 =?UTF-8?q?ers-amd64=20ve=20pahole=20ba=C4=9F=C4=B1ml=C4=B1l=C4=B1=C4=9F?=
 =?UTF-8?q?=C4=B1?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Debian 13 çekirdeğinde modül BTF adımı pahole ister; yoksa DKMS derlemesi
Error 127 ile düşer.
---
 debian/changelog | 8 ++++++++
 debian/control   | 2 +-
 2 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 45fdb64..2b4e12a 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+eta-touchdrv (0.5.1+bacak1) stable; urgency=medium
+
+  * BacakOS: depend on linux-headers-amd64 and pahole so DKMS can build the
+    modules on Debian 13 (its kbuild runs pahole for module BTF) and after
+    kernel upgrades.
+
+ -- Anadolu Panteri <bilgi@anadolupanteri.org.tr>  Sun, 18 Oct 2026 02:00:00 +0300
+
 eta-touchdrv (0.5.1) stable; urgency=medium
 
   * Fix StartLimitIntervalSec location in service file
diff --git a/debian/control b/debian/control
index 83a65de..51e58f5 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,7 @@ Homepage: https://git.pardus.org.tr
 
 Package: eta-touchdrv
 Architecture: amd64
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, linux-headers-amd64, pahole
 Description: Non-HID touchscreen drivers for Fatih IWBs
  These are drivers for non-hid 2-camera and 4-camera touchscreen sensors
  of Fatih Interactive White Boards. Kernel modules are open-source, but
-- 
2.47.3

